apps/condo/domains/miniapp/schema/B2CAppProperty.js

```diff
@@ -37,37 +52,47 @@ const B2CAppProperty = new GQLListSchema('B2CAppProperty', {
                 update: false,
             },
         },
-        address: {
-            schemaDoc: 'Property address. Must match the address lookup service with case accuracy',
-            type: Text,
-            isRequired: true,
-            hooks: {
-                resolveInput: ({ resolvedData, fieldPath }) => {
-                    if (typeof resolvedData[fieldPath] === 'string') {
-                        resolvedData[fieldPath] = resolvedData[fieldPath].toLowerCase()
-                    }
-                    return resolvedData[fieldPath]
-                },
-                validateInput: async ({ resolvedData, fieldPath, addFieldValidationError }) => {
-                    const inputAddress = resolvedData[fieldPath]
-                    if (!inputAddress) {
-                        return addFieldValidationError(INCORRECT_ADDRESS_ERROR)
-                    }
-                    // TODO(DOMA-3250) Fix this mess after migration to new address-suggestion service
-                    const suggestions = await getAddressSuggestions(inputAddress, 1)
-                    const suggestionAddress = get(suggestions, ['0', 'value'], '')
-                    const suggestionHouseType = get(suggestions, ['0', 'data', 'house_type_full'])
-                    if (!VALID_HOUSE_TYPES.includes(suggestionHouseType)) {
-                        return addFieldValidationError(`${INCORRECT_HOUSE_TYPE_ERROR}. Valid values are: [${VALID_HOUSE_TYPES.join(', ')}]`)
-                    }
-                    if (suggestionAddress.toLowerCase() !== inputAddress.toLowerCase()) {
-                        return addFieldValidationError(INCORRECT_ADDRESS_ERROR)
-                    }
+    },
+    plugins: [
+        uuided(),
+        addressService({
+            resolveAddressFields: ({ addressFields }) => {
+                if (isEmpty(addressFields)) return addressFields
+
+                const { address, ...restAddressFields } = addressFields
+                // NOTE: address string in the lower case is required for address + app unique constraint
+                return {
+                    address: address.toLowerCase(),
+                    ...restAddressFields,
+                }
+            },
+            fieldsHooks: {
+                address: {
+                    validateInput: ({ resolvedData, fieldPath, context }) => {
+                        const inputAddress = resolvedData[fieldPath]
```

Comment:
`resolvedData['address']` already contains a value from the address service.

Below you check whether the input data matches the data from the address service:
`if (suggestionAddress.toLowerCase() !== inputAddress.toLowerCase())`

But since the value in `resolvedData` was updated in the plugin, it will always be true. And the input values will not be checked